zebra/zebra_dplane.h

```diff
@@ -763,6 +763,9 @@ struct zebra_dplane_ctx *dplane_provider_dequeue_in_ctx(
 int dplane_provider_dequeue_in_list(struct zebra_dplane_provider *prov,
 				    struct dplane_ctx_q *listp);
 
+/* Current completed work queue length */
+uint32_t dplane_provider_out_ctx_queue_len(struct zebra_dplane_provider *prov);
```

Comment:
I'm not against apis, but this info is already being accessed in a show command. a) can this use a  'const' pointer? b) do you think you need this one specific api, for just one of the six or seven plugin counters?

Reply:
`struct zebra_dplane_provider` cannot be de-referenced outside zebra_dplane.c, so we require an accessor of some sort.
Originally I had `bool dplane_provider_out_ctx_queue_is_empty(...)` but changed it to be more generic, would that be preferrable?
Good point re. const, I've changed this.

Reply:
Oops - so much for 'const', sorry about that.